src/Components/Artwork/ShelfArtwork.tsx

```diff
@@ -17,6 +18,7 @@ export interface ShelfArtworkProps
   lazyLoad?: boolean
   onClick?: () => void
   width?: number[]
+  savedListId?: string
```

Comment:
The `savedListId` prop will never be used since the `ShelfArtwork` component is not used on the `collector-profile/saves2' page. For this reason we can delete this